charts/consul/values.yaml

```
@@ -776,7 +776,7 @@ server:
   # storage classes, the PersistentVolumeClaims would need to be manually created.
   # A `null` value will use the Kubernetes cluster's default StorageClass. If a default
   # StorageClass does not exist, you will need to create one.
-  # Refer to the [Read/Write Tuning](https://www.consul.io/docs/install/performance#read-write-tuning) 
+  # Refer to the [Read/Write Tuning](/docs/install/performance#read-write-tuning) 
```

Will this link work?

---

It should generate something like 

```
 Refer to the [Read/Write Tuning](/consul/docs/install/performance#read-write-tuning)
     section of the Server Performance Requirements documentation for considerations
```

which does work from mdx and is the convention we use in our docs.

---

that makes sense, but I was thinking people aren't looking at a `values.yaml` file through the docs site which is why it probably had the hostname included

---

What's interesting is that this pattern seems to be already be used in https://github.com/hashicorp/consul-k8s/blob/main/charts/consul/values.yaml#L2321 and https://github.com/hashicorp/consul-k8s/blob/64e8011be5c1eaf7b32ff6d697d6f9fc71cb3699/charts/consul/values.yaml#L303. 

I need to defer to the Consul docs team as to what might be the best recommendation here.

---

FWIW this isn't something I have a strong opinion on. Was asking more out of curiosity.

---

can we use the absolute links? this updates it to the latest absolute link

```suggestion
  # Refer to the [Read/Write Tuning](https://developer.hashicorp.com/consul/docs/install/performance#read-write-tuning) 
```